health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -2414,6 +2414,7 @@ async function processAfterPersist(request: any, actionInUrl: any) {
         localizationMap
       );
     }
+    delete request.body?.boundariesCombined;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)